integreat_cms/cms/views/push_notifications/push_notification_form_view.py

```diff
@@ -96,24 +93,26 @@ def get(self, request, *args, **kwargs):
         push_notification_form = PushNotificationForm(instance=push_notification)
 
         num_languages = len(region.active_languages)
-        PNTFormset = modelformset_factory(
-            PushNotificationTranslation,
+        PushNotificationTranslationFormSet = inlineformset_factory(
+            parent_model=PushNotification,
+            model=PushNotificationTranslation,
             form=PushNotificationTranslationForm,
-            min_num=1,  # Require at least the first form to be valid
+            min_num=num_languages,
             max_num=num_languages,
-            extra=num_languages - push_notification_translations.count(),
         )
         existing_languages = push_notification.languages if push_notification else []
-        pnt_formset = PNTFormset(
-            # Add queryset for all translations which exist already
-            queryset=push_notification_translations,
+        pnt_formset = PushNotificationTranslationFormSet(
+            # The push notification instance to which we want to create translations
+            instance=push_notification,
             # Add initial data for all languages which do not yet have a translation
             initial=[
                 {"language": language}
                 for language in region.active_languages
                 if language not in existing_languages
             ],
         )
+        # Make title of default language required
+        pnt_formset[0].fields["title"].required = True
```

Comment:
If the title field is no longer required, we need to make sure to require at least one title - otherwise, the form can just be saved with no input at all.